third-party/java/aosp/README.txt

```diff
@@ -1,13 +1,16 @@
-Android SDK (4.2.1_r1 branch) code for Manifest Merging
+manifest-merger 25.2.0
 
 == Steps
 
-1) Downloaded the Android SDK source code, follow http://source.android.com/source/downloading.html
-(use branch android-4.2.1_r1)
-2) Copied java class com.android.manifmerger.ManifestMerger and all its dependencies to the aosp/
-module.
+1) Downloaded following sources from Maven central:
+com.android.tools.build:manifest-merger:25.2.0
+com.android.tools:common:25.2.0
+com.android.tools:annotations:25.2.0
+com.android.tools:sdk-common:25.2.0
+com.android.tools:sdklib:25.2.0
+com.android.tools.layoutlib:layoutlib-api:25.2.0
+2) Copied com.android.manifmerger.ManifestMerger2 and all required dependencies to the aosp/ module.
 3) In the Android SDK's source code manifmerger, sdklib and utils are separate modules. These are
 copied into a single module which alters certain package paths. For example,
-com.android.SDKConstants becomes com.android.common.SDKConstants.
-4) Added support for re-ordering <activity-alias> so they always go after <activity> elements.
-5) Added support for merging <meta-data> elements of <application>.
\ No newline at end of file
+com.android.SDKConstants becomes com.android.common.SDKConstants. This is done to avoid conflicts
```

Comment:
Renaming SDKConstants is gross.  What libraries does this conflict with?  Should we update them?  This hack is fine for now, but you should file a task to clean it up.